common/protocols/src/main/proto/app/coronawarn/server/common/protocols/internal/submission_payload.proto

```diff
@@ -6,4 +6,5 @@ import "app/coronawarn/server/common/protocols/external/exposurenotification/tem
 
 message SubmissionPayload {
   repeated app.coronawarn.server.common.protocols.external.exposurenotification.TemporaryExposureKey keys = 1;
-}
\ No newline at end of file
+  string padding = 2;
```

Comment:
Should we maybe have a size constraint for this somewhere?

Reply:
Consider padding the HTTP package with something like a `cwa-padding` header. This wouldn't require any changes on the server side and could be implemented 100% on the mobile client side.

Reply:
If we put the padding into the proto, consider `byte[]` instead of `string` (reduced parsing overhead for data that is thrown away anyways).

Reply:
Let's keep the proto change as it is automatically downwards compatible and more explicit.

In terms of byte vs string: I guess you are right on the receiving side - should also be OK on the sender side. So I'd be fine with it.

For the length restriction: I don't think that this is available in standard protobuf - I think we should just restrict the maximum body size for the tomcat configuration - that way the potential attack will be rejected already at webserver level, and does not hit Spring/Controller/protobuf deserialization.

Reply:
As discussed, we limit the submission request size to 100KB